src/translations/en.js

```diff
@@ -1259,6 +1259,9 @@ module.exports = {
 		"yourCalendars_label": "Your calendars",
 		"yourFolders_action": "YOUR FOLDERS",
 		"yourMessage_label": "Your message",
-		"you_label": "You"
+		"you_label": "You",
+		"warning_title": "Warning",
```

Comment:
I think that three translation keys is overkill to be honest, I would add one

Reply:
Ok, we could replace "warning" with the existing "error" although this is not the same

> @vaf-hub are translations in Phrase now?

No, they are not yet in Phrase.

Reply:
I don't know if we even need a title there and I would merge the other two, what do you think?

Reply:
sounds good